.env.example

```
@@ -7,6 +7,23 @@ POSTGRES_PORT=5432
 # API
 API_PORT=8000
 
+# Keycloak (dev)
+KEYCLOAK_PORT=8080
+KEYCLOAK_ADMIN=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KEYCLOAK_ADMIN key should go before the KEYCLOAK_PORT key

(UnorderedKey)